e2e/specs/st_tooltips.spec.js

```diff
@@ -79,4 +79,187 @@ describe("tooltips on widgets", () => {
     // two sliders, st.slider and st.select_slider
     cy.get(`.stSlider .stTooltipIcon`).should("have.length", 2);
   });
+
```

Comment:
nit: would recommend grouping these `it` blocks into a new `describe` block so that the test descriptions are a bit less redundant and it's clearer which tests are similar.